api-presigned.go

```diff
@@ -118,24 +106,36 @@ func (c Client) PresignedPostPolicy(p *PostPolicy) (map[string]string, error) {
 	}
 
 	// Add date policy.
-	p.addNewPolicy(policyCondition{
+	err = p.addNewPolicy(policyCondition{
```

Comment:
when you have only err (and or temp variable) returned, better to evalaute if here..